lighthouse-core/gather/gatherers/accessibility.js

```diff
@@ -143,3 +156,4 @@ class Accessibility extends FRGatherer {
 }
 
 module.exports = Accessibility;
+module.exports._runA11yChecksInTestMode = () => runA11yChecks(true);
```

Comment:
future esmodule snipe: easier later if you just make this a method on the gatherer.